lttoolbox/ustring.cc

```diff
@@ -0,0 +1,61 @@
+#include "ustring.h"
+
+#include <stdexcept>
+#include <unicode/unistr.h>
+
+using namespace icu;
+
+void
+u_fputs(const UString& str, UFILE* output)
+{
+  u_fputs(str.c_str(), output);
+}
+
+int
+stoi(const UString& str)
+{
+  int ret;
+  int c = u_sscanf(str.c_str(), "%d", &ret);
+  if (c != 1) {
+    throw std::invalid_argument();
+  }
+  return ret;
+}
+
+double
+stod(const UString& str)
+{
+  double ret;
+  int c = u_sscanf(str.c_str(), "%f", &ret);
+  if (c != 1) {
+    throw std::invalid_argument();
+  }
+  return ret;
+}
+
+UString
+to_ustring(const char* s)
```

Comment:
These 3 functions can be optimized significantly with utfcpp which is already part of the repo.